playbooks/allocate_pubcloud.yml

```diff
@@ -163,6 +163,13 @@
   remote_user: root
   gather_facts: False
   tasks:
+    # To assist with debugging any problems if there
```

Comment:
Looks like this came in from a bad rebase?

Reply:
Dunno what happened here - will need help to get out of this pickle.